src/pybind/mgr/cephadm/templates/services/nvmeof/ceph-nvmeof.conf.j2

```diff
@@ -0,0 +1,30 @@
+# {{ cephadm_managed }}
+[gateway]
+name = {{ name }}
+group = {{ spec.group }}
+addr = {{ addr }}
+port = {{ port }}
+enable_auth = {{ spec.enable_auth }}
+state_update_notify = True
+state_update_interval_sec = 5
+
+[ceph]
+pool = {{ spec.pool }}
+config_file = /etc/ceph/ceph.conf
+
+[mtls]
+server_key = {{ spec.server_key }}
+client_key = {{ spec.client_key }}
+server_cert = {{ spec.server_cert }}
+client_cert = {{ spec.client_cert }}
+
+[spdk]
+tgt_path = {{ spec.tgt_path }}
+rpc_socket = {{ rpc_socket }}
+timeout = {{ spec.timeout }}
+log_level = {{ log_level }}
+conn_retries = {{ spec.conn_retries }}
+transports = {{ spec.transports }}
+{% if tgt_cmd_extra_args %}
+tgt_cmd_extra_args = {{ tgt_cmd_extra_args }}
```

Comment:
how are these used? I see we're not actually inserting them into the command to run nvme-of inside the container. Is the process capable of looking at this config file and picking up the args at runtime?

Reply:
@adk3798 fixed, I added support to pass extra args to the spdk binary (that runs inside the container daemon).